### PR TITLE
GitHub releases should include commit message

### PR DESCRIPTION
**Pull request by fareedmohammed**
https://eaflood.atlassian.net/browse/PSD-524

### CODE REVIEW

File: vars/buildDotNetCore.groovy [hunk 1]
```diff
@@ -93,7 +93,8 @@ void call(Map config=[:]) {
           withCredentials([
             string(credentialsId: 'github-auth-token', variable: 'gitToken')
           ]) {
-            release.trigger(tag, repoName, tag, gitToken)
+            def commitMessage = utils.getCommitMessage()
```
Comment (johnwatson484) [inline vars/buildDotNetCore.groovy:96]: Following @FayToward's PR then all files in the `var` folder should use correct types as opposed to `def`.  So this `def` should be `String`

File: vars/buildNodeJs.groovy [hunk 2]
```diff
@@ -109,7 +109,8 @@ void call(Map config=[:]) {
           withCredentials([
             string(credentialsId: 'github-auth-token', variable: 'gitToken')
           ]) {
-            release.trigger(tag, repoName, tag, gitToken)
+            def commitMessage = utils.getCommitMessage()
```
Comment (johnwatson484) [inline vars/buildNodeJs.groovy:112]: Should be `String`

File: src/uk/gov/defra/ffc/Release.groovy [hunk 3]
```diff
@@ -44,16 +44,18 @@ class Release implements Serializable {
       ctx.deleteDir()
     }
   }
-
+
```
Comment (johnwatson484) [inline src/uk/gov/defra/ffc/Release.groovy:47]: Sneaky whitespace

**Review by johnwatson484 — COMMENTED** [review]
Looks good.  Just minor suggestions

File: vars/buildDotNetCore.groovy (hunk 1 above)
Comment (fareedmohammed) [reply vars/buildDotNetCore.groovy:96]: Done.

File: vars/buildNodeJs.groovy (hunk 2 above)
Comment (fareedmohammed) [reply vars/buildNodeJs.groovy:112]: Done.

File: src/uk/gov/defra/ffc/Release.groovy (hunk 3 above)
Comment (fareedmohammed) [reply src/uk/gov/defra/ffc/Release.groovy]: Done

**Review by johnwatson484 — APPROVED** [review]

**Pull request merged**